week-2/G-conditionals-2/exercise-2.js

```diff
@@ -8,7 +8,9 @@
 */
 
 function studentPassed(grade) {
-
+  if (grade >= 50) return "pass";
+  else grade < 50;
```

Comment:
Same problem here as in the previous exercise, it's not a big problem, it just needs a bit more attention. You clearly understand the theory of conditionals but need more help with writing the actual code for it. Otherwise it would cause you problems later on when conditionals become more complicated.